packages/sync/src/class-users.php

```diff
@@ -44,13 +43,23 @@ class Users {
 	 * @todo Eventually, connection needs to be instantiated at the top level in the sync package.
 	 */
 	public static function init() {
-		self::$connection = new Jetpack_Connection();
+		add_action( 'jetpack_loaded', array( __CLASS__, 'on_jetpack_loaded' ) );
+	}
+
+	/**
+	 * Runs on Jetpack being ready to load its packages.
```

Comment:
Hooking this callback to the `jetpack_loaded` action also makes the Sync package more dependent on Jetpack, since the `jetpack_loaded` action fires in the Jetpack class.

Reply:
Do you mean that some of the configuration and initialization burden lies on the Jetpack code, rather than on the Sync code? Yes, it's a concern, but we must figure out a way to have Sync be configurable so that the consuming package actually decides what and how it initializes. Was that what you meant?

Reply:
Yeah, that’s what I meant. I see what you mean about moving configuration to the consuming plugin. Thanks for explaining that!